client/post-editor/editor-ground-control/index.jsx

```diff
@@ -1,113 +1,109 @@
-	siteUtils = require( 'lib/site/utils' ),
-	PostListFetcher = require( 'components/post-list-fetcher' ),
-	stats = require( 'lib/posts/stats' );
-
+import Card from 'components/card';
+import Gridicon from 'gridicons';
+import Popover from 'components/popover';
+import Site from 'blocks/site';
+import postUtils from 'lib/posts/utils';
+import siteUtils from 'lib/site/utils';
+import PostListFetcher from 'components/post-list-fetcher';
+import { recordEvent, recordStat } from 'lib/posts/stats';
 import AsyncLoad from 'components/async-load';
 import EditorPublishButton, { getPublishButtonStatus } from 'post-editor/editor-publish-button';
 import Button from 'components/button';
 import EditorPostType from 'post-editor/editor-post-type';
-import classNames from 'classnames';
 
-export default React.createClass( {
-	displayName: 'EditorGroundControl',
-
-	propTypes: {
-		hasContent: React.PropTypes.bool,
-		isConfirmationSidebarEnabled: React.PropTypes.bool,
-		isDirty: React.PropTypes.bool,
-		isSaveBlocked: React.PropTypes.bool,
-		isPublishing: React.PropTypes.bool,
-		isSaving: React.PropTypes.bool,
-		onPreview: React.PropTypes.func,
-		onPublish: React.PropTypes.func,
-		onSave: React.PropTypes.func,
-		onSaveDraft: React.PropTypes.func,
-		onMoreInfoAboutEmailVerify: React.PropTypes.func,
-		post: React.PropTypes.object,
-		savedPost: React.PropTypes.object,
-		setPostDate: React.PropTypes.func,
-		site: React.PropTypes.object,
-		user: React.PropTypes.object,
-		userUtils: React.PropTypes.object,
-		toggleSidebar: React.PropTypes.func,
-		type: React.PropTypes.string
-	},
-
-	mixins: [ PureRenderMixin ],
-
-	getDefaultProps: function() {
-		return {
-			hasContent: false,
-			isConfirmationSidebarEnabled: true,
-			isDirty: false,
-			isSaveBlocked: false,
-			isPublishing: false,
-			isSaving: false,
-			onPublish: noop,
-			onSaveDraft: noop,
-			post: null,
-			savedPost: null,
-			site: {},
-			user: null,
-			userUtils: null,
-			setPostDate: noop
-		};
-	},
-
-	componentDidMount: function() {
+export class EditorGroundControl extends PureComponent {
+	static propTypes = {
+		hasContent: PropTypes.bool,
+		isConfirmationSidebarEnabled: PropTypes.bool,
+		isDirty: PropTypes.bool,
+		isSaveBlocked: PropTypes.bool,
+		isPublishing: PropTypes.bool,
+		isSaving: PropTypes.bool,
+		moment: PropTypes.func,
+		onPreview: PropTypes.func,
+		onPublish: PropTypes.func,
+		onSave: PropTypes.func,
+		onSaveDraft: PropTypes.func,
+		onMoreInfoAboutEmailVerify: PropTypes.func,
+		post: PropTypes.object,
+		savedPost: PropTypes.object,
+		setPostDate: PropTypes.func,
+		site: PropTypes.object,
+		user: PropTypes.object,
+		userUtils: PropTypes.object,
+		toggleSidebar: PropTypes.func,
+		translate: PropTypes.func,
+		type: PropTypes.string
+	};
+
+	static defaultProps = {
+		hasContent: false,
+		isConfirmationSidebarEnabled: true,
+		isDirty: false,
+		isSaveBlocked: false,
+		isPublishing: false,
+		isSaving: false,
+		moment,
```

Comment:
Is it still needed as a prop? Tests don't seem to care, nor do parent components.